python/dask_cudf/dask_cudf/backends.py

```diff
@@ -40,16 +40,15 @@ def concat_cudf(
     from cudf.core.column import column, CategoricalColumn, StringColumn
     import rmm
     import cudf._lib as libcudf
-    from cudf.core.buffer import Buffer
 
     def _string_safe_hash(df):
         frame = df.copy(deep=False)
         for col in frame.columns:
             if isinstance(frame[col]._column, StringColumn):
                 out_dev_arr = rmm.device_array(len(frame), dtype="int32")
                 ptr = libcudf.cudf.get_ctype_ptr(out_dev_arr)
```

Comment:
use column constructor and get the pointer via Buffer here

Reply:
Done